src/components/projekti/suunnitteluvaihe/EsitettavatYhteystiedot.tsx

```diff
@@ -63,6 +76,12 @@ export default function EsitettavatYhteystiedot({ vuorovaikutusnro }: Props): Re
               )
             </p>
           ))}
+          {vuorovaikutusYhteysHenkilot.map((yhteystieto, index) => (
+            <p style={{ margin: 0 }} key={index}>
+              {yhteystieto.nimi}, puh. {yhteystieto.puhelinnumero},{" "}
+              {yhteystieto.email ? replace(yhteystieto.email, "@", "[at]") : ""} ({yhteystieto.organisaatio})
```

Comment:
virkamiespuolella näitä email-osoitteita ei ole tarvinnut maskata, eli ui-kuvissakin ihan @-merkillä